### ISSUE TITLE
FAB over undo button

### ISSUE THREAD

**Issue by AidanRB** [federicoiosue/Omni-Notes#219]
Undo buttons on deletion dialogs are blocked by the FAB whenever it's visible, perhaps changing the draw order would fix that.

**Comment by federicoiosue** [comment]
I can't reproduce the issue.
Here you have the right behavior, the one I see on my emulators and devices: https://drive.google.com/file/d/0B0nF1sdYwc1lRFZaRGpPYXlTNkE/view?usp=sharing

**Comment by AidanRB** [comment]
The update fixed it, thanks!

On Mon, Nov 2, 2015 at 5:45 PM Federico Iosue <notifications@github.com>
wrote:

> I can't reproduce the issue.
> Here you have the right behavior, the one I see on my emulators and
> devices:
> https://drive.google.com/file/d/0B0nF1sdYwc1lRFZaRGpPYXlTNkE/view?usp=sharing
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/federicoiosue/Omni-Notes/issues/219#issuecomment-153181780>
> .
>
-- 


    --Aidan

**Comment by federicoiosue** [comment]
Actually I never had it before too!

**Comment by federicoiosue** [comment]
Duplicate of #183
